afanasy/src/watch/item.h

```diff
@@ -4,11 +4,13 @@
 
 #include <QItemDelegate>
 #include <QStyleOption>
+#include <QMenu>
 
 class MainWidget;
 
-class Item
+class Item : public QObject
```

Comment:
I had to make `Item` a subclass of `QObject` to be able to have it host SLOTs. Hope this does not break any fundamental design choice.